### PR TITLE
f19 + f45 16pft fsurdat/landuse files to namelist_defaults_ctsm + Makefile

### LINKED ISSUE #2791: Add f19 16pft 1850 and Hist to list of surface dataset resolutions (for the PPE work)
To be consistent with the PPE work we need f19 datasets for 1850 and Historical transient for 16pfts. These datasets will first be used for the f19 spinup needed.

@olyson @djk2120 @linniahawkins 

Definition of done:
- [x] Create the datasets
- [x] rimport them
- [x] Add them to namelist_defaults
- [x] Add them for automatic creation in the mksurfdata_esmf Makefile

### PR DESCRIPTION
**Pull request by slevis-lmwg**
### Description of changes
Same as title.

### Specific notes

Contributors other than yourself, if any:

CTSM Issues Fixed (include github issue #):
Fixes #2791 
Completes most of #2810 (not sure whether I am adding the test in FATES)

Are answers expected to change (and if so in what way)?
No.

Any User Interface Changes (namelist or namelist defaults changes)?
New fsurdat/landuse files added to the defaults.

Does this create a need to change or add documentation? Did you do so?
No.

Testing performed, if any:
On derecho
PASS Tested the addition of the new fsurdat/landuse files to the Makefile with `make global-hist-1850-f19`
PASS Tested the addition of the new fsurdat/landuse files to the Makefile with `make global-hist-1850-f45`
PASS ./build-namelist_test.pl
PASS python testing 
PASS clm_pymods

### CODE REVIEW

**Comment by slevis-lmwg** [comment]
Rebase to cesm3_0_beta04_changes.

File: bld/namelist_files/namelist_defaults_ctsm.xml [hunk 1]
```diff
@@ -1766,22 +1766,13 @@ lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_mpasa15_hist_2000_16pfts_c240908.nc</f
...
 lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_1.9x2.5_hist_2000_78pfts_c240908.nc</fsurdat>
-<fsurdat hgrid="10x15"        sim_year="2000" use_crop=".true." use_vichydro=".true.">
-lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_10x15_hist_2000_78pfts_c240908.nc</fsurdat>
-<fsurdat hgrid="10x15"        sim_year="2000" use_crop=".true." use_vichydro=".false.">
+<fsurdat hgrid="10x15"     sim_year="2000" use_crop=".true.">
 lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_10x15_hist_2000_78pfts_c240908.nc</fsurdat>
-<fsurdat hgrid="4x5"          sim_year="2000" use_crop=".true." use_vichydro=".true.">
-lnd/clm2/surfdata_esmf/ctsm5.3.0/surfdata_4x5_hist_2000_78pfts_c240908.nc</fsurdat>
-<fsurdat hgrid="4x5"          sim_year="2000" use_crop=".true." use_vichydro=".false.">
+<fsurdat hgrid="4x5"       sim_year="2000" use_crop=".true.">
```
Comment (slevis-lmwg) [inline bld/namelist_files/namelist_defaults_ctsm.xml:1775]: Link to vichydro issue

**Comment by slevis-lmwg** [comment]
Adding the scope of #2810 to this PR

**Review by ekluzek — APPROVED** [review]
This looks good to me.

**Pull request merged**